odd-collector/odd_collector/adapters/postgresql/tests/test_map_table.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import odd_models
+
+
+@pytest.mark.parametrize(
+    "table_fixture, expected_data_entity_type",
+    [
+        ("table", odd_models.DataEntityType.TABLE),
+        ("table_with_vector_column", odd_models.DataEntityType.VECTOR_STORE),
+    ]
+)
+def test_map_table(postgresql_generator, table_fixture, expected_data_entity_type, request):
+    from odd_collector.adapters.postgresql.mappers.tables import map_table
+
+    table = request.getfixturevalue(table_fixture)
+
+    data_entity = map_table(postgresql_generator, table)
+    assert isinstance(data_entity, odd_models.DataEntity)
+    assert hasattr(data_entity, "type")
```

Comment:
```suggestion
```

I don't think that we need that kind of assertions. They are quite redundant.
```
isinstance(data_entity, odd_models.DataEntity)
```
There is no way that  `map_table` returns anything but DataEntity dataclass instance. So basically this assertion is always true despite arguments. So we can remove this.
```
assert hasattr(data_entity, "type")
```
Similar situation here. `type` is required property of DataEntity so `hasattr` in this case is always true. So we can remove this as well.
The same situation with `views`